<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCustomerDialog</class>
 <widget class="QDialog" name="NewCustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый клиент</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>244</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="SurnameLayout">
      <item>
       <widget class="QLabel" name="SurnameLabel">
        <property name="text">
         <string>Фамилия:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SurnameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="NameLayout">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PatronomicLayout">
      <item>
       <widget class="QLabel" name="PatronomicLabel">
        <property name="text">
         <string>Отчество:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PatronomicEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="AddressLayout">
      <item>
       <widget class="QLabel" name="AddressLabel">
        <property name="text">
         <string>Адрес:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AddressEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="EmailLayout">
      <item>
       <widget class="QLabel" name="EmailLabel">
        <property name="text">
         <string>E-mail:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmailEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PhoneCodeLayout">
      <item>
       <widget class="QLabel" name="PhoneCodeLabel">
        <property name="text">
         <string>Код города:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PhoneCodeEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PhoneNumberLayout">
      <item>
       <widget class="QLabel" name="PhoneNumberLabel">
        <property name="text">
         <string>Телефонный номер:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PhoneNumberEdit">
        <property name="minimumSize">
         <size>
          <width>122</width>
          <height>0</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>NewCustomerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>NewCustomerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
